### ISSUE TITLE
-Wtypedef-redefinition in drivers/md/bcache/btree.h

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#948]
```
00:21:51 In file included from drivers/md/bcache/alloc.c:65:
00:21:51 drivers/md/bcache/btree.h:401:14: warning: redefinition of typedef 'btree_map_keys_fn' is a C11 feature [-Wtypedef-redefinition]
00:21:51 typedef int (btree_map_keys_fn)(struct btree_op *op, struct btree *b,
00:21:51              ^
```

**Comment by nathanchance** [comment]
This has been fixed in the maintainer's tree since it was reported by 0day a couple of days ago. It just doesn't look like the patch has made it into -next yet:

https://git.kernel.org/colyli/linux-bcache/c/48a8de529652b26359a2c8cac01cc4e4da85818c

**Comment by nathanchance** [comment]
Fixed in next-20200326: https://git.kernel.org/next/linux-next/c/5ae3a2c03d1f5b33f53ce2ba2e57773fc8b35128
